how_to_submit_a_pr_-_quick_guide.md

```diff
@@ -0,0 +1,83 @@
+# How to submit a Pull Request - Quick guide
+
+
+## Objective
+
+After reading, you will know how to submit Pull Request (PR) for code review.
+
+## Prerequisite
+
+* I Understand the project goal and what is its main purpose.
+* I know who is the team lead and account manager.
+
+
+
+## The Flow
```

Comment:
This may seem a 'cold' document that is only about the flow but we have an opportunity to explain why these steps are important. Where possible, I think we should add explanations.